src/main/resources/schemas/sscs_daily_list.json

```diff
@@ -0,0 +1,414 @@
+          "title": "JOH Known As",
+          "type": "string",
+          "examples": [
+            "judge"
+          ],
+          "default": "",
+          "pattern": "^.*$"
+        }
+      }
+    }
+  },
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Root",
+  "type": "object",
+  "required": [
+    "document",
+    "venue",
+    "courtLists"
+  ],
+  "properties": {
+    "document": {
+      "$id": "#root/document",
+      "title": "document",
+      "type": "object",
+      "required": [
+        "publicationDate"
+      ],
+      "properties": {
+        "publicationDate": {
+          "$id": "#root/document/publicationDate",
+          "title": "Publication date",
+          "description": "The date + time the list was published",
+          "type": "string",
+          "examples": [
+            "2016-09-13T23:30:52.123Z"
+          ]
+        }
+      }
+    },
+    "venue": {
+      "$id": "#root/venue",
+      "title": "venue",
+      "type": "object",
+      "required": [
+        "venueName"
+      ],
+      "properties": {
+        "venueName": {
+          "$id": "#root/venue/venueName",
+          "description": "Full venue name",
+          "title": "Venue Name",
+          "type": "string",
+          "default": "",
+          "examples": [
+            "Preston"
+          ],
+          "pattern": "^.*$"
+        }
+      }
+    },
+    "courtLists": {
+      "$id": "#root/courtLists",
+      "title": "Court Lists",
+      "type": "array",
+      "items": {
+        "$id": "#root/courtLists/courtList",
+        "title": "Court list",
+        "type": "object",
+        "required": [
+          "courtHouse"
+        ],
+        "properties": {
+          "courtHouse": {
+            "$id": "#root/courtLists/courtHouse",
+            "title": "Court House",
+            "description": "The court house that owns the court rooms",
+            "type": "object",
+            "required": [
+              "courtHouseName",
+              "courtHouseContact",
+              "courtRoom"
+            ],
+            "properties": {
+              "courtHouseName": {
+                "$id": "#root/courtLists/courtList/courtHouse/courtHouseName",
+                "description": "Name of the court house",
+                "title": "Court House Name",
+                "type": "string",
+                "default": "",
+                "examples": [
+                  "PRESTON"
+                ],
+                "pattern": "^.*$"
+              },
+              "courtHouseContact": {
+                "$id": "#root/courtLists/courtList/courtHouse/courtHouseContact",
+                "description": "The contact for the court house",
+                "title": "Court House Contact",
+                "$ref": "#/definitions/venueContact"
+              },
+              "courtRoom": {
+                "$id": "#root/courtLists/courtRoom",
```

Comment:
I'm wondering if court room name is needed here. Worth double checking

Reply:
Added in :)